doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -976,6 +979,16 @@ Common Application Framework (CAF)
   * Updated the dependencies of the :kconfig:option:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option.
     The option can be enabled even when the Bluetooth controller is not enabled as part of the application that uses :ref:`caf_ble_state`.
 
+* :c:struct:`ble_peer_event`:
```

Comment:
We do not usually refer to the structures on this level of the changelog. Is this related to a specific CAF module?